### DISCUSSION TITLE
Stopped: Maximum iterations exceeded!

### DISCUSSION THREAD

**Discussion by rhw2008** [TransformerOptimus/SuperAGI#698] [Q&A]
Hello, 
I have set the max iteration to 100 and still SuperAGI did not have the time to finish everything. I've rerun it 3 times. I have added to many goals and instructions. Do you think that it is the cause. Should I rerun it several times more?  Why does SuperAGI not continue where it stopped on a new run? 
thanks

**Comment by ghost** [comment]
I have the same errors. Didnt tried thereafter. "Stopped: Maximum iterations exceeded!"

**Reply by joshknopp** [reply]
Are all 100 iterations valid? I evidently created an infinite loop by adding a goal and an instruction that were very similar to each other; my config maxed it out at 25 but I'm certain it would have happily chugged along to 100.

**Comment by neelayan7** [comment]
Is this happening on app.superagi.com?
We've noticed this issue and currently solving it.

**Reply by kamalsprasad** [reply]
Yes, for me it's the app.superagi.com
This was a few days ago. Haven't tried since.

**Reply by seahawksaregood** [reply]
can there maybe be options to go over 100 iterations?
